[16.0][MIG] l10n_es_aeat_mod592: Migration to 16.0

arantxa-s73 opened the pull request:
Module migrated to version 16.0

Commit:
[ADD] l10n_es_aeat_mod592: New module

Co-authored-by: Nicolás Ramos

christian-ramos-tecnativa committed:
[IMP] l10n_es_aeat_mod592: Misc changes

- Correciones básicas
- Búsqueda registros acquisiciones y dedducciones
- Intentar contemplar casos de fabricantes
- Traducciones y registros adquirientes
- Correciones v14
- Quitar Excel y correiones campos csv

Co-authored-by: javierjcf <javier@comunitea.com>

victoralmau committed:
[IMP] l10n_es_aeat_mod592: New module

Changes done:
- Adding specific groups to view acquirer/manufacturers
- Code refactoring
- Sequence compatibility
- Reports
- Remove report_csv dependency
- Add search option to template fields
- Show the fields in product.template if there are no variants.
- Add tests

TT48472

Commit:
[UPD] Update l10n_es_aeat_mod592.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_aeat_mod592
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_aeat_mod592/

arantxa-s73 committed:
[IMP] l10n_es_aeat_mod592): pre-commit stuff

arantxa-s73 committed:
[MIG] l10n_es_aeat_mod592: Migration to 16.0

victoralmau committed:
[FIX] l10n_es_aeat_mod592: Compare if no weight with the correct decimal precision

TT52290

victoralmau committed:
[IMP] l10n_es_aeat_mod592: Definir digits a los campos kgs, no_recycling_kgs, product_uom_qty

TT52290

victoralmau committed:
[IMP] l10n_es_aeat_mod592: Definir datos correctos en el excel que se exporta

Cambios hechos:
- Se añade el concepto y el product_description al método _get_csv_report_info() del mixin (necesario en ambos)
- Se añade el campo product_description al mixin (necesario en ambos)
- Los valores de Kilogramos y Kilogramos No Reciclados se deben definir con , como separador decimal
- Definir el valor del campo Fecha Hecho Contabilizado en formato: DD-MM-YYYY
- Adquiriente: Se define como Descripción Producto la descripción del producto en lugar de fiscal_acquirer
- Adquiriente: Se añade la columna Régimen fiscal (fiscal_acquirer)  después de la descripción de producto
- Adquiriente: Se mueven los campo Prov./Dest. al lugar adecuado, antes de Kilogramos

TT52986

victoralmau committed:
[IMP] l10n_es_aeat_mod592: Posibilidad de exportar csv

TT52986

dalonsod commented:
@arantxa-s73 could make a rebase with `origin/16.0` and push again?

dalonsod commented:
Only code review, according to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0 I've only make the following non-blocking suggested improvements:
* Is it time to move maturity to `Production/Stable`?
* You'd consider adding `unaccent=False` for some field that can be used across searches.

Commit:
[UPD] Update l10n_es_aeat_mod592.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_aeat_mod592
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_aeat_mod592/

Commit:
[UPD] Update l10n_es_aeat_mod592.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_aeat_mod592
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_aeat_mod592/

pedrobaeza commented:
@arantxa-s73 puedes revisar que aquí faltarían commits de 15.0? También revisa los comentarios de David.

/ocabot migration l10n_es_aeat_mod592

arantxa-s73 commented:
@pedrobaeza hecho

pedrobaeza reviewed (approved):
Los archivos de setup están añadidos en el commit de migración en lugar del de pre-commit changes, pero bueno, no es grave como en la siguiente al traer el historial, no se tendrá en cuenta:

/ocabot merge nobump
